configs/cms-xcache/config.d/50-cms-xcache-paths.cfg

```diff
@@ -0,0 +1,22 @@
+#
+# Configure filesystem paths for the CMS XCache server.
+#
+# **********************************************************************
+# * WARNING: DO NOT EDIT THIS FILE.  IT WILL BE OVERWRITTEN ON UPGRADE *
+# **********************************************************************
+#
+# This file is part of the CMS XCache Daemon
+
+# The directory on local disk where files will be cached
+oss.localroot /xcache-root
```

Comment:
Wait, why is this not customizable? Should be `oss.localroot $(rootdir)`, no?

Reply:
Done